hugegraph-api/src/main/java/org/apache/hugegraph/metrics/MetricsUtil.java

```diff
@@ -24,12 +24,45 @@
 import com.codahale.metrics.Histogram;
 import com.codahale.metrics.Meter;
 import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Snapshot;
 import com.codahale.metrics.Timer;
 
 public class MetricsUtil {
 
-    private static final MetricRegistry REGISTRY =
-                                        MetricManager.INSTANCE.getRegistry();
+    public static final String METRICS_PATH_TOTAL_COUNTER = "TOTAL_COUNTER";
```

Comment:
does these name will be showed to users? if yes please rename to total_counter style?